docs/OTLabels/validation/modelvalidation.md

```diff
@@ -0,0 +1,234 @@
+| ![Scene 5 TIDE](assets/plots/scene_5/tide_errors_diagram_scene_5.png)          | ![Scene 5 TIDE Night](assets/plots/scene_5/tide_errors_diagram_scene_5_night.png)          |
+| ![Scene 5 TIDE Stacked](assets/plots/scene_5/tide_stacked_diagram_scene_5.png) | ![Scene 5 TIDE Stacked Night](assets/plots/scene_5/tide_stacked_diagram_scene_5_night.png) |
+
+Scene 5 is captured at two different times, namely at day and night.
+The only difference to scene 3 is that scene 5 captures a rural road instead of a four way junction.
+Unsurprisingly, the models performed tremendously better on the dataset taken during the day compared to the one taken at night when looking at the mAP diagram.
+
+It is not surprising that all models evaluated on the nighttime dataset have a high **Missed Ground Truth Error** (Miss) which is due to the worse light conditions.
+What is standing out is the fact that the models evaluated on the day dataset also have a high miss error rate.
+To get behind the a possible reason for this we should have quick peek in the ground truth dataset:
+
+|                                                                    |                                                                      |
+| ------------------------------------------------------------------ | -------------------------------------------------------------------- |
+| ![Scene Small Object 1](assets/scenes/scene_5_small_objects_1.jpg) | ![Scene 5 Small Object 2](assets/scenes/scene_5_small_objects_2.jpg) |
+
+Objects or vehicles emerging from the road in the upper left corner of the image makes it harder for the models to detect and assign them to the correct class.
+his is reflected in the missed ground truth error and the localization error.
+Thus the further the objects move to the right area of the image, the better they are for the models to detect.
+
+## Conclusion and Future Work
+
+We have evaluated the `YOLOv5s`, `YOLOv5m`, `YOLOv5x`, and the `YOLOv5l` object detection models on a custom dataset consisting of videos capturing different scenes of traffic junctions and roads.
+Looking at mAP and TIDE metrics gave us insight to how the YOLOv5 models performed on our custom dataset and we can come to the conclusion that a `YOLOv5m`, `YOLOv5l` and `YOLOv5x` models performed far better than the `YOLOv5s`.
+But there is not a big increase in performance in terms of object detection upon choosing the `YOLOv5x` over the `YOLOv5m` or `YOLOv5l` model.
+
+Another important aspect to look into in the future is how much time the models under discussion take to detect the images.
+Depending on the use case and resources at hand, choosing the `YOLOv5x` model that might need much more time to finish the detection might not be suitable and taking `YOLOv5m` or `YOLOv5l` model might be the better choice.
+
+We also learned that how and where a camera is set up to capture a scene also influences the ability of the models to detect objects.
```

Comment:
... and where the timestamp was put ;)

Reply:
Added in latest commits :)

Reply:
And yes, there are plans to ignore and discard bounding boxes that appear in irrelevant parts such as those close to the borders of an image.

For more information see issue: OpenTrafficCam/OTValidate#24 :)

Reply:
Yes, just a joke regarding the time stamp :) (But maybe we should better put em on the top left or top right, @frunika?)